docs/05-reference/api/create-stream.mdx

```diff
@@ -67,3 +68,72 @@ https://livepeer.studio/api/stream
     {other asset object keys}
 }
 ```
+
+## Add access control policy to stream
+
+Create a stream with a playback policy.
```

Comment:
Which one is this guide about? Maybe `Create / Add` in the title?

Actually, maybe this is a better title than the current one.